setup.cfg

```diff
@@ -5,16 +5,3 @@ all_files  = 1
 
 [upload_sphinx]
 upload-dir = docs/_build/html
-
-[metadata]
-long_description = file: README.rst
-long_description_content_type = text/x-rst; charset=UTF-8
-
-[aliases]
-test = pytest
-
-[tool:pytest]
-addopts = -vv -r a
```

Comment:
Looks like this got lost somewhere :) I'll see where is the best place to add it back.

Reply:
Yes sorry, here is the doc:
https://docs.pytest.org/en/7.1.x/reference/customize.html#pyproject-toml

Reply:
It's been added back to https://github.com/sbraz/pymediainfo/tree/wheels
Let me know if you see anything that needs fixing in my branche. I squashed your commits a little and removed yet unused bits from pyproject.

Reply:
I had a look at your branch, the `MANIFEST.in` and `setup.cfg` files should be removed, they are not used anymore.

Reply:
otherwise it looks good to me :)